frontend/pages/index.tsx

```diff
@@ -92,6 +100,31 @@ export default function Home(): JSX.Element {
           );
         })}
       </div>
+
+      <Flex direction="column" align="center" width="lg">
+        <Heading mb={2} align="center" colorScheme="gray">
+          {formatMessage(
+            { id: "heroHeading" },
+            {
+              span: function renderSpan(...parts) {
+                return (
+                  <Box as="span" color="blue.500">
+                    {parts.join("")}
+                  </Box>
+                );
+              },
+            }
+          )}
+        </Heading>
+        <Text
+          mb={8}
+          align="center"
+          color={useColorModeValue("gray.600", "gray.400")}
+        >
+          {formatMessage({ id: "heroText" })}
```

Comment:
Suggestion: Can we add these ids to the `messages` object above? It should help centralize all of our ids